src/config/drive.rs

```diff
@@ -56,9 +69,9 @@ impl<'d> Drive<'d> {
         self.part_uuid.as_deref()
     }
 
-    /// Host level path for the guest drive.
-    pub fn path_on_host(&self) -> &Path {
-        &self.path_on_host
+    /// Absolute path for the guest drive.
```

Comment:
That's the same as "Host level path for the guest drive.", which I copied from upstream fc docs.

Reply:
it's a different field now (`path`)
updated a comment a bit